Dashboard/extract.py

```diff
@@ -0,0 +1,119 @@
+"Extract Script: Pulls product information from ASOS webspage"
+import json
+import logging
+import requests
+from bs4 import BeautifulSoup
+
+DEFAULT_TIMEOUT = 30
+
+
+def get_product_page(url: str, headers: dict) -> str | None:
```

Comment:
What if `url` is not of type `str` and `headers` is not of type `dict`? Will the exception below catch these?